embrace-android-delivery/src/main/kotlin/io/embrace/android/embracesdk/internal/delivery/StoredTelemetryMetadata.kt

```diff
@@ -0,0 +1,38 @@
+package io.embrace.android.embracesdk.internal.delivery
+
+import kotlin.Result.Companion.failure
+
+/**
+ * Metadata about the telemetry payload.
+ *
+ * This information is encoded in the filename.
+ */
+class StoredTelemetryMetadata(
```

Comment:
For future proofing, do we want to leave room to add additional tokens, or put in some sort of thing that allows us to version?

Reply:
Prefixing the filename with `v1` feels like a sensible addition. I don't think it makes sense to add additional tokens now as we won't know what the format will look like, but my thinking is that if the version prefix can be renamed then we can parse the filename differently without too much hassle.

Reply:
If it's all the same, I'd prefer a post-fix, so "${timestamp}_${type.description}_${uuid}_v1.json"?